js/ui/animation.js

```diff
@@ -54,14 +54,23 @@ var Animation = new Lang.Class({
         this._isPlaying = false;
     },
 
+    _getCurrentFrame: function() {
+        return this._frames[this._frameIndex];
```

Comment:
You always end up using `this._getCurrentFrame().frame`, so I'd rather have this function return that than always having to declare a variable and get the member from it.

Reply:
Ah! Then I missed that. Sorry.